.gitignore

```diff
@@ -210,3 +210,4 @@ _dependentPackages/
 
 # Fake
 .fake/
+/windowsforms/Sudoku/VB/.editorconfig
```

Comment:
Why this is needed ?
```suggestion
```

Reply:
To enforce PascalCase it shouldn't be here it should be in Samples Root where the C# one is but the VB section in it is almost empty.

Reply:
There is already one in the root of samples repo.

By the way, your last commit changed private fields from _camelCase to _PascalCase which isn't correct.

My comment was to use PascalCase for functions. private fields should remain _camelCase.

Reply:
Think I put everything back the way you want it.